src/partials/projects/projects.html

```diff
@@ -1 +1,166 @@
-<section class="projects">PROJECTS</section>
+<section class="projects">
+    <div class="swiper-container container">
+        <div class="projects-header">
```

Comment:
Try to avoid unnecessary <div> they are not needed and only add extra load to the html
I suggest removing the <div> and leaving only the <h2>
Also review the base.css file, there are already a lot of ready-made styles for text so that we don’t duplicate them in our files.
In this case, for h2 you can use the class section-title